onnx/common/tensor.h

```diff
@@ -62,6 +65,15 @@ struct Tensor final {
     return sizes_;
   }
 
+  int64_t size_from_dim(int dim) const {
+    static std::multiplies<int64_t> mul{};
```

Comment:
Does this actually same a copy each time we call this function?

Reply:
I put it as function static, so there should be only one copy got instantiated

Reply:
Yeah, it's inlined: https://godbolt.org/g/iGp6QB

:)

Reply:
wow good point! updated